src/pages/Settings/components/ModalForm.tsx

```diff
@@ -156,7 +171,21 @@ export const SettingsModalForm: React.FC<{
           <ProFormText width="lg" name="value" label={<FormattedMessage id="value" />} />
         )}
         {type === 'json' && (
-          <ProFormTextArea width="lg" name="value" label={<FormattedMessage id="value" />} />
+          <>
+            <ProFormTextArea
+              width="lg"
+              name="value"
+              label={<FormattedMessage id="value" />}
+              tooltip={`${intl.formatMessage({
+                id: 'example_json',
+              })}: 
+              {"name":"John", "age":30, "city":["New York","Warsaw"]}`}
+            />
+            {jsonValue && <ReactJson src={jsonValue} />}
```

Comment:
have a look https://procomponents.ant.design/en-US/components/form#proform

it's better to use built-in form methods 

```jsx
  {type === 'json' && <ProForm.Item noStyle shouldUpdate>
    {(form) => <ReactJson src={form.getFieldValue("value")} /> }
  </ProForm.Item>}
``` 

should work, then you can get rid of `setState`s